.github/workflows/xcm-tests.yml

```diff
@@ -5,7 +5,7 @@ on:
     - cron: '5 * * * *' # Run hourly
   push:
     branches:
-      - xcm-tests
+      - master
```

Comment:
I'm not sure what this does but otherwise lgtm

Reply:
> I'm not sure what this does but otherwise lgtm

This makes it so the test are also run when we push to `master` branch (instead of `xcm-tests`)